VAMobile/src/screens/BenefitsScreen/ClaimsScreen/ClaimsHistoryScreen/ClaimsHistoryScreen.tsx

```diff
@@ -122,18 +122,24 @@ function ClaimsHistoryScreen({ navigation }: IClaimsHistoryScreen) {
       backLabelOnPress={navigation.goBack}
       title={title}
       testID="claimsHistoryID">
-      {loadingClaimsAndAppealsList || loadingUserAuthorizedServices ? (
+      {!claimsNotInDowntime && !appealsNotInDowntime ? (
+        <ErrorComponent screenID={ScreenIDTypesConstants.CLAIMS_HISTORY_SCREEN_ID} />
+      ) : loadingClaimsAndAppealsList || loadingUserAuthorizedServices ? (
         <LoadingComponent text={t('claimsAndAppeals.loadingClaimsAndAppeals')} />
-      ) : claimsAndAppealsListError ||
-        getUserAuthorizedServicesError ||
-        (!claimsNotInDowntime && !appealsNotInDowntime) ? (
+      ) : getUserAuthorizedServicesError ? (
         <ErrorComponent
           onTryAgain={fetchInfoAgain}
           screenID={ScreenIDTypesConstants.CLAIMS_HISTORY_SCREEN_ID}
-          error={claimsAndAppealsListError}
+          error={getUserAuthorizedServicesError}
```

Comment:
We shouldn't be displaying the error component when the authorized services endpoint has an error, because if that endpoint were to fail, we don't want to block a user from using features in the app. We (Global) had a similar discussion for some HSP work, and decided if that endpoint fails, we should still allow the user to attempt to access features

Reply:
interesting, I will make those changes then

Reply:
I guess my issue now is that if that endpoint fails, then they would technically be displayed the not authorized screen due to !userAuthorizedServices?.militaryServiceHistory ?